### PR TITLE
AddToCartTest was added

### PR DESCRIPTION
**Pull request by IrynaKu**
AddToCartTest was added. Pages needed for this test were also added.

### CODE REVIEW

File: TestFramework/src/main/java/org/pages/AllLaptopsPage.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import org.base.UI.BasicPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class AllLaptopsPage extends BasicPage {
+
+    public WebElement getFirstAddButton(){
+        List<WebElement>laptopsList = baseDriver.findElements(By.xpath("//a[contains(@class, 'js-add-to-cart-button')]"));
```
Comment (Nikitskin) [inline TestFramework/src/main/java/org/pages/AllLaptopsPage.java]: Make spaces between List <WebElement> laptoplist

File: TestFramework/src/main/java/org/pages/AllLaptopsPage.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+import org.base.UI.BasicPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class AllLaptopsPage extends BasicPage {
+
+    public WebElement getFirstAddButton(){
```
Comment (Nikitskin) [inline TestFramework/src/main/java/org/pages/AllLaptopsPage.java]: make List<WebElement> as result

File: TestFramework/src/main/java/org/pages/AllLaptopsPage.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package org.pages;
+
+import org.base.UI.BasicPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+public class AllLaptopsPage extends BasicPage {
```
Comment (Nikitskin) [inline TestFramework/src/main/java/org/pages/AllLaptopsPage.java]: rename it to product page

File: TestFramework/src/main/java/org/pages/CategoryLaptop.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package org.pages;
+
+import org.base.UI.BasicPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class CategoryLaptop extends BasicPage {
+    public WebElement getAllLaptops(){
```
Comment (Nikitskin) [inline TestFramework/src/main/java/org/pages/CategoryLaptop.java]: Name correctly getAllLaptopsFilter

File: TestFramework/src/main/java/org/pages/CategoryLaptopDesktopMonitor.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
+package org.pages;
+
+import org.base.UI.BasicPage;
+import org.graalvm.compiler.phases.BasePhase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class CategoryLaptopDesktopMonitor extends BasicPage {
+
+    public WebElement getLaptops(){
```
Comment (Nikitskin) [inline TestFramework/src/main/java/org/pages/CategoryLaptopDesktopMonitor.java]: also filter?

File: TestFramework/src/main/java/org/pages/MainPage.java [hunk 6]
```diff
@@ -3,8 +3,17 @@
 import org.base.UI.BasicPage;
 import org.base.UI.WebElementWrapper;
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
 
 public class MainPage extends BasicPage{
 
-    public WebElementWrapper ComputersAndTablets = new WebElementWrapper(baseDriver.findElement(By.xpath("//a[@href='/computers-tablets']")));
+
+    public WebElement getComputersAndTablets() {
```
Comment (Nikitskin) [inline TestFramework/src/main/java/org/pages/MainPage.java]: link

File: TestFramework/src/test/java/org/testframework/coolblue/AddToCartTest.java [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+        WebElement acceptCookieButton = driver.findElement(By.name("accept_cookie"));
+        acceptCookieButton.click();
+        pageStorage.mainPage.getComputersAndTablets().click();
+        pageStorage.mainPage.getComputersAndTabletsLink().click();
+        pageStorage.laptopDesktopMonitorCategoryPage.getLaptops().click();
+        pageStorage.laptopCategoryPage.getAllLaptops().click();
+        pageStorage.allLaptopsPage.getFirstAddButton().click();
+
+        Assert.assertTrue(pageStorage.bijnaVanJouPage.getCartButton().isDisplayed());
+
```
Comment (Nikitskin) [inline TestFramework/src/test/java/org/testframework/coolblue/AddToCartTest.java]: remove all spaces

File: TestFramework/src/test/java/org/testframework/coolblue/AddToCartTest.java [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+import org.openqa.selenium.interactions.Actions;
+import org.pages.MainPage;
+import org.testframework.coolblue.common.BaseTest;
+
+public class AddToCartTest extends BaseTest {
+
+    @Test
+
+    public void addToCartTest(){
+        driver.get(Configuration.getProperty("url"));
```
Comment (Nikitskin) [inline TestFramework/src/test/java/org/testframework/coolblue/AddToCartTest.java:28]: add comments general comments

File: TestFramework/src/test/java/org/testframework/coolblue/AddToCartTest.java [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+        WebElement acceptCookieButton = driver.findElement(By.name("accept_cookie"));
+        acceptCookieButton.click();
+        pageStorage.mainPage.getComputersAndTablets().click();
+        pageStorage.mainPage.getComputersAndTabletsLink().click();
+        pageStorage.laptopDesktopMonitorCategoryPage.getLaptops().click();
+        pageStorage.laptopCategoryPage.getAllLaptops().click();
+        pageStorage.allLaptopsPage.getFirstAddButton().click();
+
+        Assert.assertTrue(pageStorage.bijnaVanJouPage.getCartButton().isDisplayed());
```
Comment (Nikitskin) [inline TestFramework/src/test/java/org/testframework/coolblue/AddToCartTest.java]: add comments to assert

File: TestFramework/src/main/java/org/pages/AllLaptopsPage.java (hunk 1 above)
Comment (IrynaKu) [reply TestFramework/src/main/java/org/pages/AllLaptopsPage.java]: done

File: TestFramework/src/main/java/org/pages/AllLaptopsPage.java (hunk 2 above)
Comment (IrynaKu) [reply TestFramework/src/main/java/org/pages/AllLaptopsPage.java]: done

File: TestFramework/src/main/java/org/pages/AllLaptopsPage.java (hunk 3 above)
Comment (IrynaKu) [reply TestFramework/src/main/java/org/pages/AllLaptopsPage.java]: done

File: TestFramework/src/main/java/org/pages/CategoryLaptop.java (hunk 4 above)
Comment (IrynaKu) [reply TestFramework/src/main/java/org/pages/CategoryLaptop.java]: done

File: TestFramework/src/main/java/org/pages/CategoryLaptopDesktopMonitor.java (hunk 5 above)
Comment (IrynaKu) [reply TestFramework/src/main/java/org/pages/CategoryLaptopDesktopMonitor.java]: done

File: TestFramework/src/main/java/org/pages/MainPage.java (hunk 6 above)
Comment (IrynaKu) [reply TestFramework/src/main/java/org/pages/MainPage.java]: done

File: TestFramework/src/test/java/org/testframework/coolblue/AddToCartTest.java (hunk 7 above)
Comment (IrynaKu) [reply TestFramework/src/test/java/org/testframework/coolblue/AddToCartTest.java]: done

File: TestFramework/src/test/java/org/testframework/coolblue/AddToCartTest.java (hunk 8 above)
Comment (IrynaKu) [reply TestFramework/src/test/java/org/testframework/coolblue/AddToCartTest.java:28]: done

File: TestFramework/src/test/java/org/testframework/coolblue/AddToCartTest.java (hunk 9 above)
Comment (IrynaKu) [reply TestFramework/src/test/java/org/testframework/coolblue/AddToCartTest.java]: done

**Review by Nikitskin — APPROVED** [review]

**Pull request merged**